app/controllers/YourIncomeCalculationController.scala

```diff
@@ -122,11 +122,13 @@ class YourIncomeCalculationController @Inject()(
       employment <- employmentService.employments(nino, year)
     } yield {
       val historicIncomeCalculationViewModel = HistoricIncomeCalculationViewModel(employment, empId, year)
-      if (printPage) {
-        Ok(views.html.print.historicIncomeCalculation(historicIncomeCalculationViewModel))
-      } else {
-        Ok(views.html.incomes.historicIncomeCalculation(historicIncomeCalculationViewModel))
+      val printablePage = views.html.print.historicIncomeCalculation(historicIncomeCalculationViewModel)
+      val incomePage = views.html.incomes.historicIncomeCalculation(historicIncomeCalculationViewModel)
+      (printPage, historicIncomeCalculationViewModel.realTimeStatus.toString) match {
```

Comment:
There could be a potential to create another view to cater for the error path. That way the view would not need to be bothered about checking for the RTI status

Reply:
Removed the checking for RTI status "TemporarilyUnavailable" in html file and using the serviceUnvailable view if RTI is down.